src/components/peopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,126 @@
+import { useParams } from 'react-router-dom';
+import classNames from 'classnames';
+import './PeopleTable.scss';
```

Comment:
according to the style guide, import styles should be at the bottom of the imports list